package.json

```diff
@@ -34,10 +34,10 @@
   },
   "devDependencies": {
     "@eslint/js": "^9.30.0",
-    "@types/eslint__js": "^9.14.0",
     "@types/node": "^24.0.8",
     "@types/which": "^3.0.4",
     "eslint": "^9.30.0",
+    "handlebars": "^4.7.8",
```

Comment:
don't you need to add handlebars to the tsup config to bundle it in the dist/*.js?